x/lockup/keeper/iterator.go

```diff
@@ -36,6 +36,7 @@ func (k Keeper) iteratorBeforeTime(ctx sdk.Context, prefix []byte, maxTime time.
 	return store.Iterator(prefix, storetypes.PrefixEndBytes(key))
 }
 
+// @TODO: confirm this existing iterator is accurate
```

Comment:
What are the TODOs for? We should create issues for any TODOs we introduce in code.